modules/components/core/org.wso2.carbon.pc.core/src/main/java/org/wso2/carbon/pc/core/internal/ProcessCenterServerHolder.java

```diff
@@ -154,7 +155,7 @@ private void deployAnalyticsDashboard() {
                 String jsonText = new String(content);
                 dashboardResource.setContent(jsonText);
                 dashboardResource.setMediaType("application/json");
```

Comment:
Fixed in https://github.com/wso2/product-pc/commit/e931bb704e9065cf57f08f61619a45a4beecb5c7